TriggerRedHatBuild.groovy

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
matter of style, and I leave it to @jflinchbaugh as to whether he wants to refactor, but generally these scripts benefit greatly from a traditional class structure and marshalling of args into a constructor. It makes clear the difference between implementation and stimulus (and also helps re-use)

Reply:
👍 will do. I had been feeling that temptation as I started extracting methods.

Reply:
wrapped into an object now.